beets/importer.py

```diff
@@ -620,8 +621,12 @@ def align_album_level_fields(self):
         """Make some album fields equal across `self.items`.
         """
         changes = {}
+        # Determine where to gather the info from for the RETAG action.
+        retag_asis = (self.choice_flag == action.RETAG and
+                      not self.items[0].artist and
+                      not self.items[0].mb_artistid)
```

Comment:
Sorry: `artist` should have been `albumartist`, in a lazy attempt to just use the same fields that the `elif` is using.

The ultimate reason is trying to find out if we get here by doing a `RETAG-based-on-asis` or 
`RETAG-based-on-candidate`, as there is no `self.match` in either case. I'm not sure if this can be achieved without further "polluting" the class (by adding a flag variable of sorts) or similar (the `match` doesn't get returned from `commands.choose_candidate()` either, just `action.RETAG`). Even if it is not bulletproof, it seemed like a good compromise to check the `mb_artistid` existence in particular as a decent indicator that we are coming from `RETAG-based-on-candidate`.